### PR TITLE
Test redirect on sign-in

### PR DESCRIPTION
**Pull request by youngkidwarrior**

### CODE REVIEW

**Comment by youngkidwarrior** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/593?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#593** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/593?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#514** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/514?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `dev`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @youngkidwarrior and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/playwright/tests/fixtures/auth/test.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { type BrowserContext, mergeTests } from '@playwright/test'
+import { type BrowserContext, type Expect, mergeTests, type Page } from '@playwright/test'
```
Comment (0xBigBoss) [inline packages/playwright/tests/fixtures/auth/test.ts:1]: is this needed?

Comment (youngkidwarrior) [reply packages/playwright/tests/fixtures/auth/test.ts:1]: they just weren't being used. you want to keep them?

**Comment by youngkidwarrior** [comment]
### Merge activity

* **Jul 15, 10:07 PM EDT**: @youngkidwarrior started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/0xsend/sendapp/593).

**Pull request merged**